src/app/trainees/reset-trainee-list/reset-trainee-list.component.html

```diff
@@ -1 +1,3 @@
-<span class="cursor-pointer" (click)="resetTraineeList()">Clear all</span>
+<button mat-button (click)="resetTraineeList()" aria-label="Clear all">
+  Clear all
```

Comment:
Seems the reset may need to be revised if underNotice=false isn't default i.e. only if